<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>562</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OPC UA MQTT Coupler Client</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Finnish" country="Finland"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="le_opcua_addr">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>opc.tcp://</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_opcua">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OPC UA Server Address:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_opcua_init">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_opcua_status">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_opcua_endp">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Endpoints</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_opcua_endpoints">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_opcua_root">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Root</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_opcua_objects">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_opcua_config">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>OpcUa Config</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>981</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabOpcUa">
     <attribute name="title">
      <string>OPC UA</string>
     </attribute>
     <widget class="QTreeWidget" name="tv_opcua">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>951</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMQTT">
     <attribute name="title">
      <string>MQTT</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_mqtt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MQTT Server Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mqtt_addr">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mqtt://</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mqtt_port">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mqtt_status">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_mqtt_init">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mqtt_port">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_mqtt_config">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>MQTT Config</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mqtt_port_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mqtt_topic">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>opcuamqtt</string>
    </property>
   </widget>
   <zorder>gb_opcua_config</zorder>
   <zorder>pb_opcua_init</zorder>
   <zorder>pb_opcua_endp</zorder>
   <zorder>tabWidget</zorder>
   <zorder>rb_opcua_root</zorder>
   <zorder>rb_opcua_objects</zorder>
   <zorder>lb_opcua</zorder>
   <zorder>le_opcua_addr</zorder>
   <zorder>cb_opcua_endpoints</zorder>
   <zorder>lb_opcua_status</zorder>
   <zorder>lb_mqtt</zorder>
   <zorder>le_mqtt_addr</zorder>
   <zorder>le_mqtt_port</zorder>
   <zorder>lb_mqtt_status</zorder>
   <zorder>pb_mqtt_init</zorder>
   <zorder>lb_mqtt_port</zorder>
   <zorder>gb_mqtt_config</zorder>
   <zorder>lb_mqtt_port_2</zorder>
   <zorder>le_mqtt_topic</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_settings"/>
    <addaction name="actionLoad_settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEncryption">
    <property name="title">
     <string>Encryption</string>
    </property>
    <addaction name="actionOpcUa_Certificate"/>
    <addaction name="actionMQTT_Certificate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEncryption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpcUa_Certificate">
   <property name="text">
    <string>OpcUa Certificate</string>
   </property>
  </action>
  <action name="actionMQTT_Certificate">
   <property name="text">
    <string>MQTT Certificate</string>
   </property>
  </action>
  <action name="actionSave_settings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionLoad_settings">
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
